recipes/openssl/ALL/conanfile.py

```diff
@@ -104,6 +104,9 @@ def build_requirements(self):
                 self.build_requires("strawberryperl/5.30.0.1")
             if not self.options.no_asm and not tools.which("nasm"):
                 self.build_requires("nasm/2.14")
+        if self._win_bash:
+            if "CONAN_BASH_PATH" not in os.environ:
```

Comment:
I think this is not the env var to check as this one is specific to Conan. I think we need other ways to check if the users have the bash in the path